src/start/input_cp2k_motion.F

```diff
@@ -1779,14 +1779,21 @@ SUBROUTINE create_pint_section(section)
                           n_var=1, default_r_val=0.5_dp)
       CALL section_add_keyword(subsection, keyword)
       CALL keyword_release(keyword)
+      CALL keyword_create(keyword, __LOCATION__, name="THERMOSTAT_ENERGY", &
+                          description="Thermostat energy for conserved quantity."// &
```

Comment:
Please add a space after the dot: `quantity. "// &`

Same below.

Reply:
Yes done. Thank you.